### PR TITLE
Issue/#594

### LINKED ISSUE #594: Publish button needs to show some progress
- Needs to show a similar bar to the preview button
- Needs to call two APIs to get a preview then fake a form download

### PR DESCRIPTION
**Pull request by darylhedley**
fixes #594 

Best to merge this #592  PR first!

### CODE REVIEW

**Comment by brian-learningpool** [comment]
+1

File: frontend/src/core/user/templates/login.hbs [hunk 1]
```diff
@@ -12,8 +12,12 @@
         <div class="login-new-user col-1-2">
 
             <div class="login-new-user-inner">
+<<<<<<< HEAD
```
Comment (dennis-learningpool) [inline frontend/src/core/user/templates/login.hbs]: conflict

File: plugins/content/subscribed/index.js [hunk 2]
```diff
@@ -0,0 +1,561 @@
+// LICENCE https://github.com/adaptlearning/adapt_authoring/blob/master/LICENSE
```
Comment (dennis-learningpool) [inline plugins/content/subscribed/index.js]: this file should not be added to core

File: plugins/content/subscribed/package.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (dennis-learningpool) [inline plugins/content/subscribed/package.json]: This file should not be added to core

**Comment by dennis-learningpool** [comment]
+1

**Pull request merged**